src/main/java/de/bonndan/nivio/model/Item.java

```diff
@@ -227,7 +236,7 @@ public boolean equals(Object o) {
 
     @Override
     public int hashCode() {
-        return Objects.hash(toString());
+        return Objects.hash(identifier);
```

Comment:
:+1:

Reply:
Shouldn't equals also check for the `identifier` now?

Reply:
I'd rather say no, because equality is given if FullyIdentifiedQualifier is equal, and this means that identifier AND group are equal (group comparison is a bit more complex, and that is encapsulated in FQI)

Reply:
I don't know enough about the internals to decide, which equality is right.

`equals() ` that uses `toString()` looked a bit unusual.
We just have to make that `hashCode()` and `equals()` fit together: If 2 objects are equal, they always *must* return the same hash code.